Doxygen escape error breaks build

Build log:

[  106s] doxygen build/botan.doxy
[  115s] /home/abuild/rpmbuild/BUILD/botan-2.2.0~b3ca3bb0/src/lib/pubkey/ed25519/ed25519_fe.h:19: error: Found unknown command `\Z' (warning treated as error, aborting now)


PR will follow soon, stay tuned.

--- a/src/lib/pubkey/ed25519/ed25519_fe.h
+++ b/src/lib/pubkey/ed25519/ed25519_fe.h
@@ -16,7 +16,7 @@
 namespace Botan {
 
 /**
-* An element of the field \Z/(2^255-19)
+* An element of the field \\Z/(2^255-19)
 */
 class FE_25519
    {

